src/PowerSystems2PRAS.jl

```diff
@@ -434,12 +336,12 @@ function generate_pras_system(
                                 PSY.get_time_series_multiple(
                                     g,
                                     s2p_meta.filter_func,
-                                    name="max_active_power",
+                                    name=component_to_formulation[g].max_active_power,
```

Comment:
can use getter function for these fields?

Reply:
Done.